### ISSUE TITLE
New prop pages - consider allowing disabling property descriptions

### ISSUE THREAD

**Issue by cartermp** [dotnet/project-system#7184]
In the new property pages, the descriptions are good to learn about what a setting does and can help with search. However, they might be seen as visually noisy for people. This is more prominent when you have your environment font turned up.

The highlighted areas indicate that we could fit 2-3 more toggles in the same view if they were turned off:

![image](https://user-images.githubusercontent.com/6309070/117344816-ac8acf80-ae5a-11eb-8fda-98d6ca523fdf.png)

Note: I personally like them there. This is feedback from dogfooding internally.

**Comment by drewnoakes** [comment]
Open question: where would such an option live?

**Comment by cartermp** [comment]
Yeah, off-handedly it'd be tools > options under projects, but it's an awfully specific thing. I figured it was worth tracking here due to the feedback, but I don't have a great idea.

**Comment by drewnoakes** [comment]
Moving to 17.1 based on feedback.
